packages/vkui/src/components/Checkbox/Checkbox.test.tsx

```diff
@@ -43,4 +45,48 @@ describe('Checkbox', () => {
     await userEvent.tab();
     expect(getDocumentBody()).toHaveFocus();
   });
+
+  it('check reset indeterminate when click to checkbox', async () => {
+    render(
+      <ConfigProvider platform={Platform.VKCOM}>
+        <Checkbox defaultIndeterminate={true}>Check</Checkbox>
+      </ConfigProvider>,
+    );
+
+    await React.act(async () => await userEvent.click(screen.getByText('Check')));
```

Comment:
```suggestion
  it('check reset indeterminate when click to checkbox', () => {
    render(
      <ConfigProvider platform={Platform.VKCOM}>
        <Checkbox defaultIndeterminate={true}>Check</Checkbox>
      </ConfigProvider>,
    );

    fireEvent.click(screen.getByText('Check'));
```
`fireEvent` из `@testing-library/react` сам `act` оборачивает

---

с текущей реализацией у меня локально чёт ругается, что не обёртнуто

<img width="320" alt="image" src="https://github.com/user-attachments/assets/f42215d4-f8e9-4f55-a354-96a3d94cb575">